samples/widgets/DateTimeRangePicker/src/GranularityModeSelector.jsx

```diff
@@ -17,8 +17,8 @@
  */
 
 import React from 'react';
-import Tabs, {Tab} from 'material-ui/Tabs';
-import { IconButton } from 'material-ui';
+import Button from 'material-ui/Button';
+import {IconButton} from 'material-ui';
```

Comment:
Instead of 2 imports, `import { Button, IconButton } from 'material-ui';`

Reply:
Fixed in bbebc2d